wordpress-mobile/gutenberg-mobile
# [iOS] Fix for a links pasting issue

## Description:

Moves forward #624 

Fixes pasting links.  The content was being updated properly, but the native component was not being refreshed right in iOS, and the change was being lost.

The format bar was showing the existence of a link but the text was just not showing it.

## Dependencies:

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/14038

## Testing:

1. Run the demo app in iOS & Android.
2. Copy a URL (`https://www.wordpress.com/).
3. Select any text.
4. Paste

Make sure the link is properly shown.
